docs/dev/test-configuration-options.md

```diff
@@ -11,21 +12,24 @@ export const Highlight = ({children, color}) => ( <span style={{
       padding: '0.2rem',
     }}>{children}</span> );
 
-This page includes a list of valid test configuration options for tests run on Sauce Labs.
+See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the code for setting the capabilities to execute a test.
 
-See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the basic code necessary for executing a test. Those settings as well as all valid parameters are described below.
+Depending on which environment you are running tests in, different options, also known as **Capabilities**, are required. There are different sets of capabilities for different environments, which can be combined. Some of these setting are required for a test to run in a given environment, while some are optional.
+* **[W3C Capabilities:](#browser-w3c-capabilities--optional)** Required for any test using Selenium or Appium to communicate with the browser. W3C capabilities are universal capabilities for any test, and are usually combined with additional capabilities
```

Comment:
We're missing the distinction between "optional" and "required"
I don't love this delineation in general, though, either, since not all of the required ones are actually required (e.g. Appium tests)....
Maybe we should merge those sections, remove the optional/required and then just be explicit about when each is required.

Reply:
I'll change it  from 'required' to 'should be set'